cpp/ql/src/Documentation/CommentedOutCode.qll

```diff
@@ -47,12 +47,27 @@ private predicate looksLikeCode(string line) {
   )
 }
 
+/**
+ * Holds if there is a preprocessor directive on the line indicated by
+ * `f` and `line`.
+ */
+private predicate preprocLine(File f, int line) {
+  exists(PreprocessorDirective pd, Location l |
```

Comment:
Shouldn't this be limited to `#else` and `#endif`? If not, please add a test with an example.

Reply:
Yes I think you're right.  Or `#elif`.  Updated.